exercises/concept/pacman-rules/.docs/hints.md

```diff
@@ -0,0 +1,12 @@
+# Hints
+
+## General
+
+- There are three boolean operators to work with boolean values: `not`, `&&`, and `||`.
+- Multiple operators can be combined in a single expression.
+
+## 1. Check if a fast attack can be made
```

Comment:
This heading seems to be unrelated to the exercise (sounds to me like it's from  "Annalyn's Infiltration")

Reply:
That's right. Edited so it applies to Pacman.